Floofbot/Services/NicknameAlertService.cs

```diff
@@ -0,0 +1,174 @@
+                        FloofDataContext _floofDb = new FloofDataContext();
+                        _floofDb.Add(new Warning
+                        {
+                            DateAdded = DateTime.Now,
+                            Forgiven = false,
+                            GuildId = badUser.Guild.Id,
+                            Moderator = msg.Author.Id,
+                            Reason = $"{moderator.Username}#{moderator.Discriminator} -> Warned by BOT for an inappropriate name",
+                            UserId = badUser.Id
+                        });
+                        _floofDb.SaveChanges();
+
+                        EmbedBuilder builder = new EmbedBuilder();
+                        builder.Title = "⚖️ Warn Notification";
+                        builder.Description = $"You have recieved a warning in {badUser.Guild.Name}";
+                        builder.AddField("Reason", "Warned by BOT for an inappropriate name");
+                        builder.Color = Color.DarkOrange;
+                        await badUser.SendMessageAsync("", false, builder.Build());
+
+                        await channel.SendMessageAsync($"Got it! I warned {badUser.Username}#{badUser.Discriminator}!");
+                    }
+                    catch (Exception ex)
+                    {
+                        await channel.SendMessageAsync("Unable to warn user. Do I have the permissions?");
+                        Log.Error("Unable to warn user for bad name: " + ex);
+                    }
+                    messageDic.Remove(msg.Id);
+                    return;
+                }
+                if (reaction.Emote.Name.Equals(kickEmoji.Name))
+                {
+                    try
+                    {
+                        //sends message to user
+                        EmbedBuilder builder = new EmbedBuilder();
+                        builder.Title = "🥾 Kick Notification";
+                        builder.Description = $"You have been Kicked from {badUser.Guild.Name}";
+                        builder.AddField("Reason", "Kicked by BOT for an inappropriate name");
+                        builder.Color = Color.DarkOrange;
+                        await badUser.SendMessageAsync("", false, builder.Build());
+
+                        await badUser.KickAsync($"{badUser.Username}#{badUser.Discriminator} -> Inappropriate Name");
+
+                        await channel.SendMessageAsync($"Got it! I kicked {badUser.Username}#{badUser.Discriminator}!");
+                    }
+                    catch (Exception ex)
+                    {
+                        await channel.SendMessageAsync("Unable to kick user. Do I have the permissions?");
+                        Log.Error("Unable to kick user for bad name: " + ex);
+                    }
+                    messageDic.Remove(msg.Id);
+                    return;
+                }
+                if (reaction.Emote.Name.Equals(removeEmoji.Name))
```

Comment:
removeNicknameEmoji makes more sense here, since I thought it was remove as in removeUser at a glance.
Also, what if the removed nickname is even more offensive?

Reply:
There will be no nickname so I dont understand what you mean by this.